BUILDING.md

```diff
@@ -115,3 +115,12 @@ the registry binary generated in the "./bin" directory:
 
 Optional [build tags](http://golang.org/pkg/go/build/) can be provided using
 the environment variable `DOCKER_BUILDTAGS`.
+
+<dl>
+<dt>noresumabledigest</dt>
+<dd>Compiles without resumable digest support</dd>
+<dt>include_gcs</dt>
+<dd>Adds support for <a href="https://cloud.google.com/storage">Google Cloud Storage</a></dd>
```

Comment:
I've noticed in some issues conversation that this seems to be broken -- just wondering if it's worth keeping it here even if that's the case, until we've fixed it

Reply:
Yup, that was the same PR that "triggered" this PR; https://github.com/distribution/distribution/pull/3676#issuecomment-1181718843

I think it's ok to keep it here, because it's _meant_ to be working, but we should have a stage in CI that also at least tries to build the binaries with these build-tags enabled.

Possibly we also need to add these build-tags to GolangCI lint (ISTR it skips linting otherwise)

Reply:
Its actually already in the CI:
https://github.com/distribution/distribution/blob/8857a1948739db3520e2fb5211979aa62c3dd5da/.github/workflows/ci.yml#L21

However, the CI runs on go 1.17. It compiles fine on go 1.17.

Reply:
Yes, I just saw that, looking into it

Reply:
OK, so I was wrong. That env var isn't doing anything because the Makefile actually wants BUILDTAGS. We should remove that env var from the yaml I guess.

Also, I used the wrong tag when checking a 1.17 version. It doesn't actually compile on 1.17. So ignore me, the code doesn't work at the moment at all.

I still think its fine to merge this, and think that part of #3676 should be to add include_gcp to the CI. We should add include_oss as well somewhere.